rust/main/agents/scraper/src/db/block.rs

```diff
@@ -84,7 +84,7 @@ impl ScraperDb {
         let models = blocks
             .map(|info| block::ActiveModel {
                 id: NotSet,
-                hash: Set(address_to_bytes(&info.hash)),
+                hash: Set(h256_to_bytes(&info.hash)),
```

Comment:
Fix of an old bug. It was exposed with predefined hashes with a lot of zeros at the front.